app/src/androidTest/java/com/android/sample/ui/profile/EditProfileTest.kt

```diff
@@ -47,14 +46,6 @@ class EditProfileScreenTest {
     navigationActions = Mockito.mock(NavigationActions::class.java)
     `when`(navigationActions.currentRoute()).thenReturn(PROFILE)
     `when`(profileViewModel.userState).thenReturn(userStateFlow)
-    doAnswer { invocation ->
```

Comment:
If removing this line makes your tests pass, seems good!

Reply:
yeah nothing to add !